src/index.js

```diff
@@ -101,14 +101,15 @@ class CssModuleFactory {
 class MiniCssExtractPlugin {
   constructor(options = {}) {
     validateOptions(schema, options, 'Mini CSS Extract Plugin');
-
+    const insert = options.insert ? options.insert.toString() : null;
```

Comment:
Why do not move that to bottom and use

```js
this.options.insert
 ? Template.asString([this.options.insert, 'insert(linkTag);'])
 : 'var head = document.getElementsByTagName("head")[0]; head.appendChild(linkTag)'
```

And avoid extra code? Because now, `insert` is always `string`

Reply:
Will do